docs/admin/settings/leo_gateway.md

```diff
@@ -131,6 +131,22 @@ And also, when size of a requested object more than `cache.cache_max_content_len
 
 ![Figure: Disk Cache Limits](../../assets/leofs-gateway-disk-cache-size.png)
 
+#### Configurations which may affect the cache behavior
+Whether or not an object is cached on LeoGateway is determined by below logics.
+
+- LeoGateway's cache feature is only available to small objects when cache.cache_disc_capacity = 0.
+- An object which size is larger than cache.cache_disc_threshold_len never be cached if cache.cache_disc_capacity = 0.
+
+How LeoGateway's cache feature works with cache related configurations is described below.
+
+- For small objects
+    - Cache happens in both cases handling a PUT request in the write through way and handling a GET in the read through way
+    - Being stored into memory if the size of an object < cache.cache_disc_threshold_len
```

Comment:
Being stored into memory if the size of an object < `cache.cache_disc_threshold_len`